R/aggregate.R

```diff
@@ -442,10 +442,37 @@ agg_udef <- function(diff, conf) {
 get_indicator_level <- function(gdf, ind) {
   gdf$level <- ""
   gdf$level_direction <- NA
+
+  round_digits <- function(level) {
+    # return 2 decimal places for levels equal 1 or greater under the assumption
+    # that integer percentages will always be used in presentations
+    if (level >= 1) {
+      return(2)
```

Comment:
Det kommer en advarsel om at denne linjen ikke testes. Jeg kan prøve å utvide testen slik at denne linjen blir tatt med?

Reply:
Det skyldes "sære" tilfeller som jeg har funnet i database, eksempelvis for traume. Jeg har ikke satt meg inn i om slike verdier kan forventes eller om de faktisk er feil...

Reply:
Fra verify:
![image](https://user-images.githubusercontent.com/1522857/120978498-d21a3a00-c774-11eb-9f85-5a5e11aa4138.png)